rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/ApplicationResource.java

```diff
@@ -315,6 +320,21 @@ private Response launch(String yaml, EntitySpec<? extends Application> spec) {
         }
     }
 
+    private void waitForStart(final Application app, Duration timeout) {
+        // without this UI shows "stopped" sometimes esp if brooklyn has just started,
+        // because app goes to stopped state if it sees unstarted children,
+        // and that gets returned too quickly, before the start has taken effect
```

Comment:
i'm not quite so down on this.  the problem is that the ui polls for status, and polls immediately after it gets confirmation that an application has been created.  since we almost always start right after creation, we now hold off on confirmation until it is started, so the ui doesn't see the misleading brief initial state transactions.

if we had real-time sockets going to the ui then there would be no need to wait for start but in a polling world this feels good to me.